templates/ocp4.3_pxeboot.j2

```diff
@@ -0,0 +1,16 @@
+DEFAULT {{ inventory_hostname }}-deploy
+PROMPT 0
+TIMEOUT 50
+TOTALTIMEOUT 6000
+ONTIMEOUT {{ inventory_hostname }}-deploy
+
+LABEL local
+        MENU LABEL (local)
+        MENU DEFAULT
+        LOCALBOOT 0
+
+LABEL {{ inventory_hostname}}-deploy
+        MENU LABEL RHCOS {{ rhcos_version }} {{ arch }}- Kickstart for {{ inventory_hostname }}
+        KERNEL http://{{ http_server_ip }}/rhcos-{{ rhcos_version }}-{{ arch }}-installer-kernel-{{ arch }}
```

Comment:
for pxe, path to kernel (and also initrd) should be local to the pxe boot server, so not over http (as network isn't yet initialized, while it is with ipxe). Reason why in our other template, we have local path and the boot server role has the files in place automatically.